src/prefect/deployments/base.py

```diff
@@ -377,21 +377,17 @@ def _copy_deployments_into_prefect_file():
             f.write(raw_deployment_file_contents)
 
 
-def _save_deployment_to_prefect_file(
+def _format_deployment_for_saving_to_prefect_yaml(
```

Comment:
I moved some of the logic from `_save_deployment_to_prefect_file` to a helper function so I could use it elsewhere